input/pagecontent/conformance.md

```diff
@@ -40,9 +40,11 @@ AU eRequesting profile elements include mandatory and *Must Support* requirement
 #### AU eRequesting Capability Statements
 The [AU eRequesting Placer CapabilityStatement](CapabilityStatement-au-erequesting-placer.html) defines the conformance requirements and expectations of an [AU eRequesting Placer](ActorDefinition-au-erequesting-actor-placer.html) actor responsible for initiating and creating diagnostic service requests. The complete list of FHIR profiles, REST API interactions, and search parameters that can be implemented by an AU eRequesting Placer are defined in this capability statement. AU eRequesting placers define their capabilities by choosing from this list based on the resource types they need to place requests. 
 
-The [AU eRequesting Server CapabilityStatement](CapabilityStatement-au-erequesting-server.html) defines the conformance requirements and expectations of an [AU eRequesting Server](ActorDefinition-au-erequesting-actor-server.html) actor responsible for accepting diagnostic service requests and making diagnostic service requests accessible. The complete list of FHIR profiles, REST API interactions, and search parameters that can be implemented by an AU eRequesting Server are defined in this capability statement. An AU eRequesting Server declares conformance to this list of capabilities based on the resource types and interactions it implements.
+The [AU eRequesting Filler CapabilityStatement](CapabilityStatement-au-erequesting-filler.html) defines the conformance requirements and expectations of an [AU eRequesting Filler](ActorDefinition-au-erequesting-actor-filler.html) actor responsible for finding and retrieving diagnostic service requests so that service providers can fulfil them. The complete list of FHIR profiles, REST API interactions, and search parameters that can be implemented by an AU eRequesting Placer are defined in this capability statement. AU eRequesting placers define their capabilities by choosing from this list based on the resource types they need to place requests. 
```

Comment:
"...implemented by an AU eRequesting Placer .." replace with Filler

Reply:
filler

Reply:
Not addressed

Reply:
true, I changed a different placer to filler in the same para.